Test/Unit/Model/Api/UniversalApiTest.php

```diff
@@ -136,17 +88,19 @@ public function orderCreation_returnsTrue()
     {
         $event = "order.create";
         $data = ['order' => 'order', 'currency' => 'currency'];
+        $createOrder = $this->createMock(CreateOrder::class);
+        $createOrder->expects(self::once())->method('execute')->with($event,
+            $data['order'],
+            $data['currency']
+        );
 
-        $this->createOrder->expects(self::once())
-            ->method('execute')
-            ->with(
-                $event, 
-                $data['order'], 
-                $data['currency']
-            )
-            ->willReturn(true);
-        
-        $this->assertTrue($this->currentMock->execute($event, $data));
+        $apiHelperProperty = new \ReflectionProperty(
+            UniversalApi::class,
+            'createOrder'
+        );
+        $apiHelperProperty->setAccessible(true);
```

Comment:
You could use in cases like this
https://github.com/BoltApp/bolt-magento2/blob/master/Test/Unit/TestHelper.php#L62

Reply:
@danac-gn I've updated the code.